File: platform-core-service/platform-core-service.Common/Models/DTOs/EntityDTO/CommunityQAPostReports/SelectCommunityQAPostReportsDTO.cs
using platform_core_service.Common.Models.DTOs.EntityDTO.CommunityContentReport;

namespace platform_core_service.Common.Models.DTOs.EntityDTO.CommunityQAPostReports
{
    public class SelectCommunityQAPostReportsDTO : BaseSelectCommunityReportDTO
    {
        public string QAPostId { get; set; } = null!;
        public SelectReportedQAPostDTO? QAPost { get; set; }
    }
}

File: frontend/src/types/community-qa-post-reports/select-community-qa-post-reports-dto.ts
import { BaseSelectCommunityReportDTO } from "../community-content-report/base-select-community-report-dto";
import { SelectReportedQAPostDTO } from "../community-content-report/select-reported-content-dto";

export interface SelectCommunityQAPostReportsDTO extends BaseSelectCommunityReportDTO {
    qaPostId: string;
    qaPost?: SelectReportedQAPostDTO | null;
}
